LICENSE

```diff
@@ -260,6 +260,9 @@ The PostgreSQL software includes:
       gpMgmt/bin/pythonSrc/ext/parse-1.8.2.tar.gz
       see licenses/LICENSE-parse.txt
 
+      gpMgmt/bin/pythonSrc/ext/ruff-0.11.5.tar.gz
+      see licenses/LICENSE-ruff.txt
+
```

Comment:
If we install `ruff` via the internet instead of bundling its tarball in our source code, no need to add the license info, just handle it like other dependencies, right?

https://infra.apache.org/licensing-howto.html#bundled-vs-non-bundled

Reply:
@tuhaihe You are correct. Let me remove the license file updates from this PR.